Sources/Amplitude/Plugins/iOS/UIKitElementInteractions.swift

```diff
@@ -115,6 +123,11 @@ extension UIGestureRecognizer {
 
         guard state == .ended, let view else { return }
 
+        // Block scroll events for `UIScrollView`.
+        if view is UIScrollView && self is UIPanGestureRecognizer {
```

Comment:
I would compare `if let scrollView = view as? UIScrollView, self === scrollView.panGestureRecognizer {`. perhaps for `pinchGestureRecognizer` as well, unless you want to capture zooms.